cmake-modules/AddTestCMocka.cmake

```diff
@@ -65,6 +65,11 @@ function(ADD_CMOCKA_TEST _TARGET_NAME)
 
     add_executable(${_TARGET_NAME} ${_add_cmocka_test_SOURCES})
 
+    # Suppress clobber warning for longjmp
+    if(CMAKE_C_COMPILER_ID MATCHES "GNU")
```

Comment:
Nice comment.

For reference:
https://cmake.org/cmake/help/v3.0/variable/CMAKE_LANG_COMPILER_ID.html
> GNU = GNU Compiler Collection (gcc.gnu.org)